src/elements/tabs/tab-label/tab-label.visual.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { html } from 'lit';
```

Comment:
I don't know if could be truly useful, but i would like to have other cases covered, eg. without amount, without icon, without both, slotted icon, slotted amount, long label etc; basically the same cases covered in stories.